@types/theme.ts

```diff
@@ -0,0 +1,20 @@
+interface Theme {
+  palette: IPalette;
```

Comment:
`palette`가 `Color` enum의 모든 color를 포함하는거라면 아래와 같이 간단하게 타이핑할 수 있을 것 같습니다.

```suggestion
  palette: Record<Color, string>;
```